xmldiff/actions.py

```diff
@@ -1,17 +1,17 @@
 from collections import namedtuple
 
 # The edit script actions used in xmldiff
-DeleteNode = namedtuple('DeleteNode', 'node')
+DeleteNode = namedtuple('DeleteNode', 'node tag')
 InsertNode = namedtuple('InsertNode', 'target tag position')
-RenameNode = namedtuple('RenameNode', 'node tag')
+RenameNode = namedtuple('RenameNode', 'node tag node_right tag_left')
```

Comment:
Same thing with the tag_left here, that information is already in the node xpath. The node_right path is new information, but I'm not sure how that is useful. That really goes for the rest of the changes as well. I'm not sure what problem this solves.

Reply:
Hi, the problem with this tool is that it does not have enough details on the diffs. It might not be a big problem for small xmls, but I have large xmls which I do not memorize every node and tag names. 

For example, all the update diff nodes only have right values and not left values. If i see a long list of diffs with just right values, then, the whole tool/diff is useless information for me without knowing what it was before. 

Therefore, I made these changes and other changes (which I may or may not implement to this tool - it includes separating xml elements with unique attribute/value ids so that you don't need to decide F or ratio_mode for diff_options and also diff output as excel so that user can filter easily) to make a new tool for my workplace, which is working very well. I thought there were other people who might feel the same and might find this useful.

Reply:
I see, you are trying to use it as a visual tool. I think in that case it's probably better to come up with some sort of output format that is more useful and clear. I don't think the actual tuples need to duplicate information.

Reply:
Might be duplicate information for these ones, but not for the Update tuples.

Reply:
Yeah, we could include the old information as well, there's already an enhancement request for that.

I'm closing this one, but if you have ideas for a nice human readable diff format I'm all ears, we don't really have one now, and I don't know how it would look.

Reply:
I created an issue for that: https://github.com/Shoobx/xmldiff/issues/53